CommonTools/RecoAlgos/interface/RecoTrackSelector.h

```diff
@@ -95,21 +83,26 @@ class RecoTrackSelector {
 	}
       }
     }
+    
     bool algo_ok = true;
     if (algorithm_.size()!=0) {
       if (std::find(algorithm_.begin(),algorithm_.end(),t.algo())==algorithm_.end()) algo_ok = false;
     }
     return
-      (t.hitPattern().trackerLayersWithMeasurement() >= minHit_ &&
+      (
+       (algo_ok & quality_ok) &&
+       t.hitPattern().numberOfValidHits() >= minHit_ &&
```

Comment:
this will cause regression for those using minHit
it is not silent as cmsRun will complain about missing minLayer (and more)